### PR TITLE
coot/dns ttl

### PR DESCRIPTION
**Pull request by coot**
- Fixed a few typos and code layout
- DNSActions: refactor code
- dns-actions: make sure TTLs are strictly greater than 0

# Description

_description of the pull request, if it fixes a particular issue it should link
the PR to a particular issue, see
[ref](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword=)_

# Checklist

- Branch
    - [ ] Commit sequence broadly makes sense
    - [ ] Commits have useful messages
    - [ ] New tests are added if needed and existing tests are updated
    - [ ] If this branch changes Consensus and has any consequences for downstream repositories or end users, said changes must be documented in [`interface-CHANGELOG.md`](../ouroboros-consensus/docs/interface-CHANGELOG.md)
    - [ ] If this branch changes Network and has any consequences for downstream repositories or end users, said changes must be documented in [`interface-CHANGELOG.md`](../docs/interface-CHANGELOG.md)
    - [ ] If serialization changes, user-facing consequences (e.g. replay from genesis) are confirmed to be intentional.
- Pull Request
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description at least containing the following information:
      - What does this PR change?
      - Why these changes were needed?
      - How does this affect downstream repositories and/or end-users?
      - Which ticket does this PR close (if any)? If it does, is it [linked](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)?
    - [ ] Reviewer requested

### CODE REVIEW

File: ouroboros-network/src/Ouroboros/Network/PeerSelection/RootPeersDNS/DNSActions.hs [hunk 1]
```diff
@@ -371,3 +373,13 @@ ioDNSActions =
...
              (Left  e6, Right r4) -> return ([e6], r4)
              (Right r6, Right r4) -> return ([], r6 <> r4)
+
+
+--
+-- Utils
+--
+
+
+fixupTTL :: DNS.TTL -> DNS.TTL
```
Comment (njd42) [inline ouroboros-network/src/Ouroboros/Network/PeerSelection/RootPeersDNS/DNSActions.hs:383]: I would suggest that we map all other positive numbers below, say, 30 to 30.

There is no reason for us to have a TTL less than that - we don't want to ever churn faster than that.

**Review by njd42 — APPROVED** [review]
LGTM - mapping small TTLs to 30 would seem sensible.

File: ouroboros-network/src/Ouroboros/Network/PeerSelection/RootPeersDNS/DNSActions.hs (hunk 1 above)
Comment (coot) [reply ouroboros-network/src/Ouroboros/Network/PeerSelection/RootPeersDNS/DNSActions.hs:383]: That was my original suggestion :)

**Comment by coot** [comment]
bors merge

**Pull request merged**